### PR TITLE
added ZeroBin

### PR DESCRIPTION
**Pull request by cavebeat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -602,6 +602,7 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
   * [Sticky Notes](http://sayakb.github.io/sticky-notes/) - A powerful open-source pastebin application for effortless code sharing. `PHP`
   * [Stikked](https://github.com/claudehohl/Stikked) - An advanced and beautiful pastebin written in `PHP` `GPLv3`
   * [tastebin](https://github.com/andreineculau/tastebin) - Updated version of hastebin plus additional features `Apache` `Nodejs`
+  * [ZeroBin](http://sebsauvage.net/wiki/doku.php?id=php:zerobin) - ZeroBin is a minimalist, opensource online pastebin/discussion board where the server has zero knowledge of hosted data. ([Demo](https://zerobin.net/), [Source Code](https://github.com/sebsauvage/ZeroBin/)) `Zlib` `JavaScript` `PHP`
```
Comment (nodiscc) [inline README.md]: PHP only, same as #449. Thanks

**Comment by cavebeat** [comment]
PHP & JS same as https://github.com/Kickball/awesome-selfhosted/pull/449#issuecomment-184890220

**Comment by Kickball** [comment]
Please update the language to be:

    `PHP/Javascript`

**Comment by Kickball** [comment]
Please see #407 as we previously declined to add zero bin due to lack of maintainer.

We can reconsider this if we can find a new maintainer.

**Comment by nodiscc** [comment]
Ah I forgot it was unmaintained. I don't know if we should include it in the current state. However there seems to be a lead to an active fork in https://github.com/sebsauvage/ZeroBin/issues/103

**Comment by cavebeat** [comment]
could be considered the upstream repo, i'll update
https://github.com/elrido/ZeroBin This branch is 290 commits ahead, 48 commits behind sebsauvage:master.

**Comment by nodiscc** [comment]
This fork looks very active and in a good shape :+1: Thanks.

**Comment by cavebeat** [comment]
without this pull request discussions, i would not have found it on my own. have to update soon. 
thanks for being picky :stuck_out_tongue:

**Comment by mro** [comment]
so do I, thx!

**Pull request merged**